custom/motioncontrollers/CustomPIDController.java

```diff
@@ -350,4 +352,64 @@ public double get() {
 			return 0;
 		}
 	}
+
+	/**
+	 * Put the PID controller to SmartDashboard for tuning.
+	 * Puts the error, setpoint, sensor value, and output.
+	 * This method adds a very small random value to everything
+	 * so that graphs show properly on SmartDashboard.
+	 * 
+	 * @param prefix
+	 *        The prefix to use when putting things on SmartDashboard.
+	 */
+	public void putToSmartDashboard(String prefix) {
```

Comment:
This should be added as an abstract function to MotionController (same for the other functions).